### PR TITLE
feat(sql-server): Update SQL Server example to use Prisma Migrate

### PR DESCRIPTION
**Pull request by 2color**

### CODE REVIEW

File: databases/sql-server/prisma/schema.prisma [hunk 1]
```diff
@@ -1,9 +1,46 @@
 generator client {
   provider        = "prisma-client-js"
-  previewFeatures = ["microsoftSqlServer", "connectOrCreate"]
+  previewFeatures = ["microsoftSqlServer", "referentialActions"]
```
Comment (thebiglabasky) [inline databases/sql-server/prisma/schema.prisma:3]: Depending on when we want to merge we might skip the microsoft feature flag but for now that's fine

**Review by thebiglabasky — COMMENTED** [review]
Looks great!
I'll try it out on my workstation and give some further feedback if I encounter anything

**Comment by janpio** [comment]
Tests failing because of: https://github.com/prisma/prisma/issues/8058

**Comment by Jolg42** [comment]
This should work with 2.29.0 so https://github.com/prisma/prisma-examples/pull/3063 needs to be merged first.

**Pull request merged**